api2/pkg/individuals/store.go

```diff
@@ -58,12 +58,32 @@ func (s *Store) Get(ctx context.Context, ID string) (*Individual, error) {
 
 func (s *Store) List(ctx context.Context, listOptions ListOptions) (*IndividualList, error) {
 
-	filter := bson.M{}
+	filter := bson.M{
+		"partyTypeIds": bson.M{
+			"$elemMatch": bson.M{
+				"$eq": partytypes.IndividualPartyType.ID,
+			},
+		},
+	}
 
 	if len(listOptions.PartyTypeIDs) > 0 {
-		filter["partyTypeIds"] = bson.M{
-			"$all": listOptions.PartyTypeIDs,
+		filter = bson.M{
+			"$and": bson.A{
+				bson.M{
+					"partyTypeIds": bson.M{
+						"$all": listOptions.PartyTypeIDs,
+					},
+				},
+				bson.M{
+					"partyTypeIds": bson.M{
+						"$elemMatch": bson.M{
+							"$eq": partytypes.IndividualPartyType.ID,
+						},
```

Comment:
```suggestion
                includesIndividualType := false
                for _, partyTypeID := listOptions.PartyTypeIds {
                  if partyTypeID == partytypes.IndividualPartyType.ID {
                    includesIndividualType = true
                  } 
                }
                if !inclduesIndividualType {
                  listOptions.PartyTypeIds = append(listOptions.PartyTypeIDs, partytypes.IndividualPartyType.ID)
                }
                filter["partyTypeIds"] = bson.M{
			"$all": listOptions.PartyTypeIDs
	         }
	         
```

Reply:
Here, you can just add IndividualPartyTypeID if it is not already in listOptions.PartyTypeIDs

Reply:
Changes made!